src/containers/News/News.module.scss

```diff
@@ -0,0 +1,58 @@
+@import "../Sass/main.scss";
+
+   h1 {
+    margin: 0px;
+    padding: 0px;
+    text-align: center;
+    font-size: $blogs-news-heading-font;
+    color: $blogs;
+    font-weight: 600;
+   }
+   .container{
+     margin-top: 30px;
+     margin-left: 100px;
+     margin-right: 100px;
+     text-align: center;
+      .subContainers{
+      display: inline-block;
+      border: 1px solid $grey;
+      width: 80%;
+      min-height: 130px;
+      margin: 30px;
+      text-align: center;
+      padding: 0px; 
+      outline: none;
+      background: $blogsBackground;
+      color: $blogs;
+      border-radius: 10px;
+      cursor: pointer;
+      position: relative;
+      word-wrap: break-word;
+      .upperSection{
+        background: $loginSignup;
+        height: 45%;
+        text-align: center;
+        border-top-left-radius: 9px;
+        p{
+          font-size: $paragraph-font;
+        }
+        h1{
```

Comment:
please do not use element selector in CSS as it searches through the whole CSS tree. try to be more specific by using class, id, attribute selectors etc.